037-WindowsVirtualDesktop/Student/05-Create-Configure-HostPools.md

```diff
@@ -0,0 +1,45 @@
+# Challenge 5: Create and configure host pools and session hosts
+
+[< Previous Challenge](./04-Create-Manage-Images.md) - **[Home](../README.md)** - [Next Challenge >](./06-Implement-Manage-FsLogix.md)
+
+## Introduction
+
+In this challenge you will Create and Configure Host Pools and Session Hosts for your environment. 
```

Comment:
Unnecessary capitalization here, this is a sentence, not a section header, only the first letter needs capitalization.